### PR TITLE
PMM-11310 playwright CLI tests for modb and help

### PR DESCRIPTION
**Pull request by vyarosh**
https://jira.percona.com/browse/PMM-11310

- https://github.com/percona/pmm-ui-tests/pull/477

### CODE REVIEW

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3049-afffb99
client docker - perconalab/pmm-client-fb:PR-3049-afffb99
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3049-afffb99.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3049-afffb99&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3049-afffb99.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3049-764fca9
client docker - perconalab/pmm-client-fb:PR-3049-764fca9
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3049-764fca9.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3049-764fca9&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3049-764fca9.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3049-c7f0bce
client docker - perconalab/pmm-client-fb:PR-3049-c7f0bce
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3049-c7f0bce.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3049-c7f0bce&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3049-c7f0bce.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3049-f0b0aca
client docker - perconalab/pmm-client-fb:PR-3049-f0b0aca
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3049-f0b0aca.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3049-f0b0aca&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3049-f0b0aca.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3049-184245a
client docker - perconalab/pmm-client-fb:PR-3049-184245a
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3049-184245a.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3049-184245a&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3049-184245a.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3049-3a0dff7
client docker - perconalab/pmm-client-fb:PR-3049-3a0dff7
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3049-3a0dff7.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3049-3a0dff7&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3049-3a0dff7.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3049-9dcbfbb
client docker - perconalab/pmm-client-fb:PR-3049-9dcbfbb
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3049-9dcbfbb.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3049-9dcbfbb&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3049-9dcbfbb.tar.gz

**Comment by vyarosh** [comment]
https://github.com/Percona-Lab/pmm-submodules/actions/runs/4136202091 - ready to go

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3049-ce9c503
client docker - perconalab/pmm-client-fb:PR-3049-ce9c503
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3049-ce9c503.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3049-ce9c503&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3049-ce9c503.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3049-a8c6b20
client docker - perconalab/pmm-client-fb:PR-3049-a8c6b20
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3049-a8c6b20.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3049-a8c6b20&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3049-a8c6b20.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3049-a940ecb
client docker - perconalab/pmm-client-fb:PR-3049-a940ecb
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3049-a940ecb.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3049-a940ecb&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3049-a940ecb.tar.gz

**Comment by vyarosh** [comment]
https://github.com/Percona-Lab/pmm-submodules/actions/runs/4140401960

**Comment by vyarosh** [comment]
https://github.com/Percona-Lab/pmm-submodules/actions/runs/4140401960

File: .github/workflows/pmm2-testsuite.yml [hunk 1]
```diff
@@ -207,9 +236,16 @@ jobs:
...
+        if: ${{ matrix.db-type == 'modb6' || matrix.db-type == 'modb5' || matrix.db-type == 'modb4.4' || matrix.db-type == 'help' }}
+        uses: actions/upload-artifact@v3
+        with:
+          name: "report-${{ matrix.db-type }}"
+          path: ./pmm-ui-tests/cli/test-report
+
       - name: Create status check
-        if: ${{ always() }}
-        uses: percona-platform/github-status-action@v1
+        if: ${{ env.SHA != 'null' }}
```
Comment (yurkovychv) [inline .github/workflows/pmm2-testsuite.yml]: please roll back to always()
and try condition `always() && env.SHA != 'null' `

File: .github/workflows/pmm2-testsuite.yml [hunk 2]
```diff
@@ -219,3 +255,16 @@ jobs:
           repository: ${{ github.repository }}
           target_url: "https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}"
           sha: ${{ env.SHA }}
+
+      - name: Create status check
```
Comment (yurkovychv) [inline .github/workflows/pmm2-testsuite.yml]: please remove this step

File: .github/workflows/pmm2-testsuite.yml (hunk 1 above)
Comment (vyarosh) [reply .github/workflows/pmm2-testsuite.yml]: Done though I've placed it into the variable conditional

File: .github/workflows/pmm2-testsuite.yml (hunk 2 above)
Comment (vyarosh) [reply .github/workflows/pmm2-testsuite.yml]: done

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3049-936662a
client docker - perconalab/pmm-client-fb:PR-3049-936662a
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3049-936662a.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3049-936662a&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3049-936662a.tar.gz

**Comment by vyarosh** [comment]
https://github.com/Percona-Lab/pmm-submodules/actions/runs/4164914728/jobs/7207197398

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3049-69a86a8
client docker - perconalab/pmm-client-fb:PR-3049-69a86a8
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3049-69a86a8.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3049-69a86a8&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3049-69a86a8.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3049-61adf53
client docker - perconalab/pmm-client-fb:PR-3049-61adf53
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3049-61adf53.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3049-61adf53&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3049-61adf53.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3049-61adf53
client docker - perconalab/pmm-client-fb:PR-3049-61adf53
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3049-61adf53.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3049-61adf53&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3049-61adf53.tar.gz

**Review by yurkovychv — APPROVED** [review]

**Pull request merged**